sampleAppKotlin/src/main/res/navigation/nav_graph.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is the navigation graph.
It looks like the screenshot on Android Studio
![Screen Shot 2021-05-18 at 16 14 01](https://user-images.githubusercontent.com/7802052/118607916-18265280-b7f4-11eb-83d5-a0db0072028b.png)